### DISCUSSION TITLE
CTRL+RightArrow no longer works for word completion

### DISCUSSION THREAD

**Discussion by Butanium** [community/community#54593] [Copilot Conversations]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/55806347/236587491-3561b5e5-f375-4d7d-9090-b1d129e59088.png)
Copilot allow `accept word` with `ctrl+right arrow`. But with the last vscode version this is in conflict with 
```json
{
  "key": "ctrl+right",
  "command": "cursorWordEndRight",
  "when": "textInputFocus"
}
```
Disabling `cursorWordEndRight` makes accept word works again.
Behavior with `cursorWordEndRight`:

https://user-images.githubusercontent.com/55806347/236587583-4eb6a351-16d4-42e2-95c4-29705bba9580.mp4

Behavior without `cursorWordEndRight`:

https://user-images.githubusercontent.com/55806347/236587683-8866ebf6-2e19-4088-90aa-e7cc0827ae1f.mp4

**Comment by Butanium** [comment]
# GitHub Copilot
v1.85.75

# Vscode
Version : 1.78.0
Validation : 252e5463d60e63238250799aef7375787f68b4ee
Date : 2023-05-03T20:09:46.685Z
Electron : 22.4.8
Chromium : 108.0.5359.215
Node.js : 16.17.1
V8 : 10.8.168.25-electron.0
OS : Linux x64 5.19.0-41-generic snap
Sandboxed : No

**Comment by kferrone** [comment]
Me too but how did you disable it?

**Reply by Butanium** [reply]
just remove the keybinding in the keyboard shortcuts
